draft-ietf-taps-interface.md

```diff
@@ -661,43 +669,47 @@ Security configuration parameters and sample usage follow:
 
 - Local identity and private keys: Used to perform private key operations and prove one's
 identity to the Remote Endpoint. (Note, if private keys are not available, e.g., since they are
-stored in HSMs, handshake callbacks MUST be used. See below for details.)
+stored in HSMs, handshake callbacks must be used. See below for details.)
 
 ~~~
 SecurityParameters.AddIdentity(identity)
 SecurityParameters.AddPrivateKey(privateKey, publicKey)
 ~~~
 
 - Supported algorithms: Used to restrict what parameters are used by underlying transport security protocols.
-When not specified, these algorithms SHOULD default to known and safe defaults for the system. Parameters include:
+When not specified, these algorithms should default to known and safe defaults for the system. Parameters include:
 ciphersuites, supported groups, and signature algorithms.
 
 ~~~
-SecurityParameters.AddSupportedGroup(22)    // secp256k1
-SecurityParameters.AddCiphersuite(0xCCA9)   // TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256
-SecurityParameters.AddSignatureAlgorithm(7) // ed25519
+SecurityParameters.AddSupportedGroup(secp256k1)
+SecurityParameters.AddCiphersuite(TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256)
+SecurityParameters.AddSignatureAlgorithm(ed25519)
 ~~~
```

Comment:
Do these things need references?  E.g. "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256" ?

Reply:
eh... in examples, it's probably okay

Reply:
I agree with Brian here, especially since these are meant to be abstract constants that each implementation could, on its on, redefine using some other constant or enum.

Reply:
ok by me then. just wanted to check